tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,41 @@
+"""Handles logging functionality."""
+
+import sys
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from inspect import getframeinfo
+from inspect import stack
+
+def log(log_priority, publisher_id, log_payload, job_id=None,
+        flow_id=None, parent_id=None, cluster_id=None,
+        message_id=None, timestamp=None, node_id=None,
+        caller=None, *args, **kwargs):
```

Comment:
no here caller is not required
remove caller, node_id, timestamp, message_id and args, kwargs